plugins/dialog/src/models.rs

```diff
@@ -65,4 +67,44 @@ pub enum MessageDialogButtons {
     OkCustom(String),
     /// 2 buttons `Ok` and `Cancel` with custom texts
     OkCancelCustom(String, String),
+    /// 3 buttons `Yes`, `No` and `Cancel` with custom texts
+    YesNoCancelCustom(String, String, String),
+}
+
+/// Result of a message dialog
+#[derive(Debug, Default, Clone, PartialEq, Eq, Deserialize, Serialize)]
+pub enum MessageDialogResult {
+    Yes,
+    No,
+    Ok,
+    #[default]
```

Comment:
We won't need this if we impl `into`

Reply:
we still need it, for when the message from kotling/swift to rust fails

Reply:
Got it